testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -919,22 +922,25 @@ impl BlockMinerThread {
     fn make_vrf_proof(&mut self) -> Option<VRFProof> {
         // if we're a mock miner, then make sure that the keychain has a keypair for the mocked VRF
         // key
+        let sortition_hash = if matches!(self.reason, MinerReason::EmptyTenure) {
+            self.burn_election_block.sortition_hash
+        } else {
+            self.burn_block.sortition_hash
+        };
```

Comment:
I'm pretty sure that this could always be set to `self.burn_election_block.sortition_hash`